wsproto/connection.py

```diff
@@ -380,8 +380,9 @@ def _process_connection_request(self, event):
         if b'sec-websocket-version' not in headers:
             return ConnectionFailed(CloseReason.PROTOCOL_ERROR,
                                     "Missing Sec-WebSocket-Version header")
-        # XX FIXME: need to check Sec-Websocket-Version, and respond with a
-        # 400 if it's not what we expect
+        if headers[b'sec-websocket-version'] != WEBSOCKET_VERSION:
```

Comment:
@Kriechi I think the RFC allows for a client to [send multiple allowed versions](https://tools.ietf.org/html/rfc6455#section-4.4), with the server allowed to respond with the one it wants to use (or 400 with a set of possible versions). I think therefore this is too strict, in this form.